inventory/modelcartlist.py

```
@@ -0,0 +1,36 @@
+from turtle import mode
```

Ruff (0.8.2)

`turtle.mode` imported but unused

Remove unused import: `turtle.mode`

(F401)